rpm/podman.spec

```diff
@@ -276,57 +225,33 @@ CGO_CFLAGS=$(echo $CGO_CFLAGS | sed 's/-specs=\/usr\/lib\/rpm\/redhat\/redhat-an
 export CGO_CFLAGS+=" -m64 -mtune=generic -fcf-protection=full"
 %endif
 
-export GO111MODULE=off
-export GOPATH=$(pwd)/_build:$(pwd)
-
-mkdir _build
-cd _build
-mkdir -p src/%{provider}.%{provider_tld}/%{project}
-ln -s ../../../../ src/%{import_path}
-cd ..
-ln -s vendor src
-
 # build date. FIXME: Makefile uses '/v2/libpod', that doesn't work here?
 LDFLAGS="-X %{import_path}/libpod/define.buildInfo=$(date +%s)"
```

Comment:
I think if you build with gomodules you now have to use the `/v4/libpod` path like the makefile does. I assume without that `podman version` will not show the built date.
But please double check with the built rpm I am not sure if there is some special rpm magic that I am missing.

Reply:
@Luap99  updated LDFLAGS to match Makefile. In future work, I'll try to reuse the Makefile as much as possible and reduce dup. Also, what's the latest on rootlessport build with BUILDTAGS. Do we need to get it in right away ?

Reply:
using the LDFLAGS like what's done now did show me the built date.

Reply:
rootlessport should not need buildtags: https://github.com/containers/podman/pull/18875/commits/f31466e7479344d9968f6d679d9c2ef5cb45a360